packages/inventory-general-info/src/CollectionCard.js

```diff
@@ -3,15 +3,24 @@ import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import LoadingCard from './LoadingCard';
 import { collectionInformationSelector } from './selectors';
+import { DateFormat } from '@redhat-cloud-services/frontend-components';
 
 const CollectionCard = ({ detailLoaded, collectionInformation, entity }) => (<LoadingCard
     title="Collection information"
     isLoading={ !detailLoaded }
     items={ [
         { title: 'Insights client', value: collectionInformation.client },
         { title: 'Egg', value: collectionInformation.egg },
-        { title: 'Last check-in', value: entity && new Date(entity.updated).toLocaleString() },
-        { title: 'Registered', value: entity && new Date(entity.created).toLocaleString() }
+        { title: 'Last check-in', value: entity && (
+            DateFormat ?
+                <DateFormat date={ entity.updated } type="onlyDate" /> :
+                new Date(entity.updated).toLocaleString()
+        ) },
+        { title: 'Registered', value: entity && (
+            DateFormat ?
```

Comment:
Just a question... is `DateFormat` loading so slow that yah gotta test for its availability before using?

Reply:
That's to check if the component is actually accessible, since it's a new-ish component and the consumer could have outdated FEC bundle. If that would happen users could have see a broken UI.